### DISCUSSION TITLE
Create a way to see the full dependents list

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/scaladex#1107] [Ideas]
Today I looked on Scaladex to try and get a full view of who all depended on a project, for example `scala-xml`. On [this page](https://index.scala-lang.org/scala/scala-xml#dependents) I can see there are 339 dependants and I can see a big chunk of them, but there is no way (from what I can see) to see the full list. It'd be great to be able to see that whole list.

<img width="418" alt="Screenshot 2022-11-11 at 10 06 31" src="https://user-images.githubusercontent.com/13974112/201305753-ab7bfb0d-3220-442b-a76f-8dd0c597edb6.png">

Maybe what I have circled could be made clickable or something?

**Comment by adpi2** [comment]
Thanks for opening this discussion. I am considering moving the project dependencies and dependents to a new tab of the project page. Thus we could have paging on them, and maybe more information on which artifacts, versions, Scala version and platforms those dependent projects depend on.

When I say a new tab, I mean this:
![image](https://user-images.githubusercontent.com/13123162/201317005-cbecb647-5a84-443e-b6d0-65b4aac8da27.png)

**Comment by dplyukhin** [comment]
I'd like this too. I think at one point it was possible to search for a project's dependents with a special query API. For example, to find all projects that depend on akka, you could search `dependencies:akka/*`. But either that feature was removed, or it changed and it's not obvious how to do it anymore from the user interface.

**Reply by adpi2** [reply]
Yes the feature was removed because it was not documented, not tested and really hard to maintain over all the refactoring that we did lately. I am not sure if we should revive it or not. Having the full list of dependent projects on the project page seems to address the need and it would be easier to maintain.

**Reply by dplyukhin** [reply]
Fair enough. FWIW, I wanted the feature so I could find the most popular projects that depend on Akka or Pekko. There isn't another way to do that with Scaladex, is there?

**Reply by adpi2** [reply]
No unfortunately there isn't a way currently. Maybe when we have the full list of dependents on the project page the user can decide how to sort them.

**Reply by adpi2** [reply]
Here is what I can extract from the database directly:
[akka-dependents.csv](https://github.com/scalacenter/scaladex/files/13974295/akka-dependents.csv)
[pekko-dependents.csv](https://github.com/scalacenter/scaladex/files/13974297/pekko-dependents.csv)

**Reply by dplyukhin** [reply]
Oh thanks! I was having trouble getting Scaladex to run on my machine (succeeded in connecting to the elasticsearch container, but not the postgres container for some reason) so this saves me a lot of time.
